ci/scripts/model_health_check.py

```
@@ -197,6 +220,42 @@ def check_embedder(model: str, api_key: str) -> tuple[int, str, str]:
+                if verbose:
+                    files = section[model]
+                    for f in sorted(set(files)):
+                        _logger.info("      - %s", f)
+
+            report[label.lower()] = report_rows
+
+        if output_file is not None:
+            write_json_report(output_file=output_file, report=report)
+
+        return 0
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)